src/cli/trn_cli_network_policy.c

```diff
@@ -321,6 +333,73 @@ int trn_cli_update_transit_network_policy_enforcement_subcmd(CLIENT *clnt, int a
 	return 0;
 }
 
+int trn_cli_update_agent_network_policy_enforcement_subcmd(CLIENT *clnt, int argc, char *argv[])
+{
+	ketopt_t om = KETOPT_INIT;
+	struct cli_conf_data_t conf;
+	cJSON *json_str = NULL;
+
+	if (trn_cli_read_conf_str(&om, argc, argv, &conf)) {
+		return -EINVAL;
+	}
+
+	char *buf = conf.conf_str;
+	json_str = trn_cli_parse_json(buf);
+	int counter = cJSON_GetArraySize(json_str);
+
+	if (json_str == NULL) {
+		return -EINVAL;
+	}
+
+	int *rc;
+	struct rpc_trn_vsip_enforce_t enforces[counter];
+	char rpc[] = "update_agent_network_policy_enforcement_1";
+
+	for (int i = 0; i < counter; i++)
+	{
+		struct rpc_trn_vsip_enforce_t enforce;
+		enforce.interface = conf.intf;
+		enforce.count = counter;
+		cJSON *policy = cJSON_GetArrayItem(json_str, i);
+
+		int err = trn_cli_parse_network_policy_enforcement(policy, &enforce);
+		if (err != 0) {
+			print_err("Error: parsing network policy enforcement config.\n");

```

Comment:
nit: print out the error detail (like returned error code) if it is meaningful

Reply:
Detailed log is in trn_cli_parse_network_policy_enforcement func in cli_common.c file